### PR TITLE
Proc creation lnx local account

### PR DESCRIPTION
**Pull request by CheraghiMilad**
### Summary of the Pull Request
Detects enumeration of local systeam accounts. This information can help adversaries determine which local accounts exist on a system to aid in follow-on behavior.
Add a new technique with new binaries and file path.

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/)

### CODE REVIEW

File: rules/linux/process_creation/proc_creation_lnx_local_account.yml [hunk 1]
```diff
@@ -28,10 +28,13 @@ detection:
             - '/tail'
             - '/vi'
             - '/vim'
+            - '/less'
+            - '/cp'
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_local_account.yml:32]: `cp` did not math the rule logic Discovery

File: rules/linux/process_creation/proc_creation_lnx_local_account.yml [hunk 2]
```diff
@@ -28,10 +28,13 @@ detection:
...
         CommandLine|contains:
             - '/etc/passwd'
             - '/etc/shadow'
             - '/etc/sudoers'
+            - '/etc/spwd.db'
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_local_account.yml:37]: I didn't know `spwd.db` , It seems that there are also `/etc/pwd.db` 
https://man.cx/pwd_mkdb(8)

Comment (CheraghiMilad) [reply rules/linux/process_creation/proc_creation_lnx_local_account.yml:37]: Yes, during the attack simulation, I observed the spwd.db file. For this reason, I only mentioned it, but we can also add the other two files, namely /etc/pwd.db and /etc/master.passwd, to this rule.

**Review by frack113 — APPROVED** [review]
LGTM

File: rules/linux/process_creation/proc_creation_lnx_local_account.yml [hunk 3]
```diff
@@ -28,10 +28,14 @@ detection:
             - '/tail'
             - '/vi'
             - '/vim'
+            - '/less'
         CommandLine|contains:
             - '/etc/passwd'
             - '/etc/shadow'
             - '/etc/sudoers'
+            - '/etc/spwd.db'
```
Comment (nasbench) [inline rules/linux/process_creation/proc_creation_lnx_local_account.yml:36]: Add references to these files

**Review by nasbench — CHANGES_REQUESTED** [review]

**Review by nasbench — APPROVED** [review]

**Pull request merged**